Bucketeer2/Sources/Public/BKTConfig.swift

```diff
@@ -0,0 +1,99 @@
+import Foundation
+
+public struct BKTConfig {
+    let apiKey: String
+    let endpoint: URL
```

Comment:
Please rename all `endpoint` variables to `apiEndpoint`.

Reply:
I think we could also rename the internal implementations.

https://github.com/ca-dp/bucketeer-ios-sdk/blob/v2/Bucketeer2/Sources/Internal/Remote/ApiClientImpl.swift#L7-L8

Reply:
Thanks! The parameters are also renamed.
https://github.com/ca-dp/bucketeer-ios-sdk/pull/86/commits/19a97cc07ef2eeb1f288b71ee36eb7d47148d152

Reply:
Can you please also rename the test files?
https://github.com/ca-dp/bucketeer-ios-sdk/blob/v2/Bucketeer2Tests/ApiClientTests.swift#L22
Maybe there are other files to be renamed.